config/management/src/validator_config.rs

```diff
@@ -43,7 +43,7 @@ impl ValidatorConfig {
         let consensus_key = ed25519_from_storage(CONSENSUS_KEY, local.as_mut())?;
         let fullnode_network_key = x25519_from_storage(FULLNODE_NETWORK_KEY, local.as_mut())?;
         let validator_network_key = x25519_from_storage(VALIDATOR_NETWORK_KEY, local.as_mut())?;
-        let owner_key = ed25519_from_storage(OWNER_KEY, local.as_mut())?;
+        let operator_key = ed25519_from_storage(OPERATOR_KEY, local.as_mut())?;
```

Comment:
Does this remove the need for signing permission on the `owner` key? Or are partners going to be using `management owner-key` as well?

Reply:
there are moving pieces, as long as we keep coordinated on the doc, we can seamlessly upgrade from milestone to milestone while minimizing the headache for partners.

I prefer we only simplify and never increase complexity -- we're not ready to simplify yet